.gitignore

```diff
@@ -41,8 +41,9 @@ Testing/IccToXml
 # Xcode deps
 Build/XCode/libtiff/*.h*
 Build/XCode/libxml/*.h*
+
 # Xcode build files
-**/build
+**/*.noindex
```

Comment:
Please keep the build path ignore part. It is used for CMake builds.

Reply:
The project root has a folder named [Build](https://github.com/InternationalColorConsortium/RefIccMAX/tree/master/Build), which then is undesirably ignored on Windows.

Reply:
Resolved in 716d3e5 by ignoring all build folders except that one.